operator/v1alpha1/register.go

```diff
@@ -32,6 +32,8 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 	metav1.AddToGroupVersion(scheme, GroupVersion)
 
 	scheme.AddKnownTypes(GroupVersion,
+		&ClusterCSIDriver{},
```

Comment:
are you supporting this API as GA?  Meaning on by default.

 If so, then this is a v1 API (supported for one year or three releases, whichever is longer).

If not, how will a  user opt-in?

Reply:
discussed this offline. Moved the API to v1.